flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -258,65 +258,21 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
     private func captureEnvelope(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
         guard let arguments = call.arguments as? [Any],
               !arguments.isEmpty,
-              let event = arguments.first as? String else {
+              let data = (arguments.first as? FlutterStandardTypedData)?.data  else {
             print("Envelope is null or empty !")
             result(FlutterError(code: "2", message: "Envelope is null or empty", details: nil))
             return
         }
-
-        do {
-            let envelope = try parseJsonEnvelope(event)
-            SentrySDK.capture(envelope)
-            result("")
-        } catch {
-            print("Cannot parse the envelope json !")
-            result(FlutterError(code: "3", message: "Cannot parse the envelope json", details: nil))
+        // We need to replace this when we have the possibility to access envelope deserialization.
+        guard let sentrySerialization = NSClassFromString("SentrySerialization") as? NSObject.Type,
+            let unmanaged = sentrySerialization.perform(NSSelectorFromString("envelopeWithData:"), with: data),
+            let envelope = unmanaged.takeUnretainedValue() as? SentryEnvelope else {
+            print("Cannot parse the envelope data")
+            result(FlutterError(code: "3", message: "Cannot parse the envelope data", details: nil))
```

Comment:
@bruno-garcia @philipphofmann can we expose a way how to deserialize an envelope out of an array of bytes? we're always using a String but now with the possibility of attachments etc this would not work great anymore